nyx_client/nyx_client/client.py

```diff
@@ -517,6 +518,30 @@ def get_data(
             for resp in resps
         ]
 
+    def get_my_data_by_name(self, name: str) -> Data:
+        """Retrieve a data based on its unique name.
```

Comment:
- "Retrieve my own data"?
- The name only unique for that organisation

Ah ok - I see you have that below but I think it might be important enough to state in the summary line :thinking: